tests/test_initial_configuration_from_server.rs

```diff
@@ -121,15 +63,17 @@ fn main() {
         "collection_id".to_string(),
     );
     let client =
-        AppConfigurationClientHttp::new(address, Box::new(MockTokenProvider {}), config_id)
+        AppConfigurationClientHttp::new(address, Box::new(common::MockTokenProvider {}), config_id)
             .unwrap();
 
+    common::wait_until_online(&client);
```

Comment:
Not for this PR, but:
Is this intended to be exposed to the user?
I guess if we query features immediately after client construction, we will get errors right?
Depending on use-case this might be unexpected by the user.

Should we make this more convenient? Ideas:
- make the wait function part of client
- wait a specific time when first interaction is requested